package.json

```diff
@@ -20,13 +20,16 @@
     "disqus-react": "^1.0.6",
     "dotenv": "^8.1.0",
     "gatsby": "^2.15.9",
+    "gatsby-image": "^2.2.20",
     "gatsby-plugin-feed": "^2.3.11",
     "gatsby-plugin-manifest": "^2.2.14",
     "gatsby-plugin-offline": "^3.0.3",
     "gatsby-plugin-react-helmet": "^3.1.6",
     "gatsby-plugin-sass": "^2.1.13",
+    "gatsby-plugin-sharp": "^2.2.25",
     "gatsby-source-contentful": "^2.1.33",
     "gatsby-source-wordpress": "^3.1.30",
+    "gatsby-transformer-sharp": "^2.2.16",
```

Comment:
Are you using all three of these packages?

Reply:
gatsby-image installs the other 2...

Reply:
Ah, cool, makes sense